app/src/features/projects/list/ProjectsListTable.tsx

```diff
@@ -0,0 +1,150 @@
+import { makeStyles, Theme } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Chip from '@material-ui/core/Chip';
+import Link from '@material-ui/core/Link';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import clsx from 'clsx';
+import { SubmitStatusChip } from 'components/chips/SubmitStatusChip';
+import { DATE_FORMAT } from 'constants/dateTimeFormats';
+import { BioHubSubmittedStatusType } from 'constants/misc';
+import { IGetDraftsListResponse } from 'interfaces/useDraftApi.interface';
+import { IGetProjectsListResponse } from 'interfaces/useProjectApi.interface';
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { getFormattedDate } from 'utils/Utils';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  projectsTable: {
+    tableLayout: 'fixed'
+  },
+  linkButton: {
+    textAlign: 'left',
+    fontWeight: 700
+  },
+  chip: {
+    minWidth: '7rem',
+    fontSize: '11px',
+    textTransform: 'uppercase'
+  },
+  chipDraft: {
+    borderColor: '#afd3ee',
+    backgroundColor: 'rgb(232, 244, 253)'
+  }
+}));
+
+export interface IProjectsListTableProps {
+  projects: IGetProjectsListResponse[];
+  drafts: IGetDraftsListResponse[];
+}
+
+const ProjectsListTable: React.FC<IProjectsListTableProps> = (props) => {
+  const classes = useStyles();
+
+  const { projects, drafts } = props;
+  const hasProjects = projects.length > 0;
+  const hasDrafts = drafts?.length > 0;
+
+  function getProjectSubmissionStatus(project: IGetProjectsListResponse): BioHubSubmittedStatusType {
+    if (project.projectSupplementaryData.has_unpublished_content) {
+      return BioHubSubmittedStatusType.UNSUBMITTED;
+    }
+    return BioHubSubmittedStatusType.SUBMITTED;
+  }
+
+  if (!hasProjects && !hasDrafts) {
+    return (
+      <TableContainer>
+        <Table className={classes.projectsTable}>
+          <TableHead>
+            <TableRow>
+              <TableCell>Name</TableCell>
+              <TableCell>Type</TableCell>
+              <TableCell>Contact Agency</TableCell>
+              <TableCell>Status</TableCell>
+              <TableCell>Start Date</TableCell>
+              <TableCell>End Date</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <TableCell colSpan={6}>
+                <Box display="flex" justifyContent="center">
+                  No Results
+                </Box>
+              </TableCell>
+            </TableRow>
+          </TableBody>
+        </Table>
+      </TableContainer>
+    );
+  } else {
```

Comment:
Can move this outside of the `else` branch

Reply:
moved